docker/lite/install_dependencies.sh

```diff
@@ -56,7 +56,8 @@ mysql56)
         libmysqlclient18
         mysql-client
         mysql-server
-        percona-xtrabackup-24
+        libcurl3
+        percona-xtrabackup
```

Comment:
What's the difference between percona-xtrabackup and percona-xtrabackup-24? In the bootstrap Dockerfile we are installing the other one.
https://github.com/vitessio/vitess/blob/44b9ec6d129fba29789f242fe444566d9c6df582/docker/bootstrap/Dockerfile.mysql56#L20

Reply:
the percona-xtrabackup package is the one built against mysql 5.6, xtrabackup-24 is built against mysql 5.7

Reply:
Then our bootstrap Dockerfile is wrong.